tests/spec/test_dagspec.py

```diff
@@ -1129,6 +1129,33 @@ def test_import_tasks_from(tmp_nbs):
     ]
 
 
+def test_import_tasks_from_empty_yaml_file(tmp_nbs):
+    with pytest.raises(ValueError) as excinfo:
+        Path('some_tasks.yaml').write_text('')
+
+        spec_d = yaml.safe_load(Path('pipeline.yaml').read_text())
+        spec_d['meta']['import_tasks_from'] = 'some_tasks.yaml'
+
+        spec = DAGSpec(spec_d)
```

Comment:
as a best practice when writing tests that evaluate error messages. it's best to only include the statement that throws the error, in this case:

```python
with pytest.raises(ValueError) as excinfo:
    DAGSpec(spec_d)
```

Please move everything the `spec = ...` statement outside the `with pytest.raises` line, and you can remove the `spec.to_dag().render()`

Reply:
Thanks @edublancas! Just pushed my changes